config/hev_config.h

```diff
@@ -0,0 +1,37 @@
+#ifdef CONFIG_TOP
+#include "proffieboard_v2_config.h"
+#define NUM_BLADES 1
+#define NUM_BUTTONS 2
+#define VOLUME 50
+const unsigned int maxLedsPerStrip = 144;
+#define CLASH_THRESHOLD_G 3.0
+//If not disabled, Armor Readout play back floats around current Armor value.
+#define DISABLE_NO_REPEAT_RANDOM
+#endif
+
+#ifdef CONFIG_PROP
+#include "../props/hev.h"
+#endif
+
+// All HALF-LIFE sfx audio can be found in the game's "sound" directory.
```

Comment:
This comment is really helpful, but I would imagine that you still have to convert sounds to 44.1khz PCM mono and edit and rename some sounds.  It would be great to have a guide that explains what needs to be done to make a functional font from the game directories.  Although, that information should probably be in the prop file, not here. (And it could be in a different PR.)